fix build when numpy is not installed

jerryzh168 opened the pull request:
Summary: att

Differential Revision: D13055635

ezyang commented on caffe2/python/pybind_state.h:
```diff
@@ -246,10 +246,11 @@ class TensorFeeder : public BlobFeederBase {
+    return tensor;
 #else
     CAFFE_THROW("Caffe2 compiled without NumPy support.");
+    return caffe2::Tensor(); // will not reach here
```
Uhhh, OK. (I feel like this shouldn't be necessary, but fine.)

jerryzh168 committed:
fix build when numpy is not installed (#13948)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/13948

att

Reviewed By: ezyang

Differential Revision: D13055635

fbshipit-source-id: af1e30b0d70ba91bafdd846d3b896b6ab6443281
